src/DqtApi/V2/Requests/UpdateTrnRequest.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using DqtApi.V2.ApiModels;
+using DqtApi.V2.Responses;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
+
+namespace DqtApi.V2.Requests
+{
+    public class UpdateTrnRequest : IRequest<UpdateTrnResult>
+    {
+        [SwaggerParameter(Description = "Id of teacher")]
+        [FromQuery(Name = "Trn")]
+        public string Trn { get; set; }
+
+        [Required]
+        public UpdateTrnRequestInitialTeacherTraining InitialTeacherTraining { get; set; }
+        [Required]
+        public UpdateTrnRequestRequestQualification Qualification { get; set; }
+    }
+
+    public class UpdateTrnRequestInitialTeacherTraining
+    {
+        public string ProviderUkprn { get; set; }
+        [Required]
+        public DateOnly ProgrammeStartDate { get; set; }
```

Comment:
Not sure how the validator can work if these properties aren't nullable.
```suggestion
        public DateOnly? ProgrammeStartDate { get; set; }
```